vagrant/provisioning/roles/arkcase-prerequisites/tasks/main.yml

```diff
@@ -55,27 +55,37 @@
       vars:
         item: arkcase
 
+- name: "Check if node is installed"
+  become: yes
+  shell: "node --version"
+  register: is_present
```

Comment:
`is_present` isn't clear ... what is present?  Maybe `node_is_present` or `node_version`